src/pages/resultsView/expression/ExpressionWrapper.tsx

```diff
@@ -338,10 +336,12 @@ export default class ExpressionWrapper extends React.Component<ExpressionWrapper
     }
 
     @computed
-    get alphabetizedStudies(){
-        return _.chain(this.props.studyMap).values().sortBy(
-            (study:CancerStudy)=>study.name
-        ).value();
+    get alphabetizedAndFilteredStudies() {
```

Comment:
@kalletlak this is good example of when filtering would not work properly unless this.studiesWithExpressionData.result was finished loading.   Is it clear that the this could not render without that being isComplete?

Reply:
Yes, this loads only after this.studiesWithExpressionData is complete